src/store/mutations.js

```diff
@@ -12,4 +13,9 @@ export default {
   importLibrary(state, payload) {
     state.models.library = payload;
   },
+  changeUnits(state, newState) {
+    _.keys(state).forEach((k) => {
+      state[k] = newState[k];
+    });
+  },
```

Comment:
@devKate should I be using something built-in instead of this? I've seen you use `$store.setState` before.

Reply:
I have no recollection of using this 😬